spec/controllers/home_controller_spec.rb

```diff
@@ -6,9 +6,16 @@
 # Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
-require 'spec_helper'
+require 'rails_helper'
 
 describe HomeController do
+  let!(:parsers) do
+    allow_any_instance_of(Partner).to receive(:update_apis)
+    allow_any_instance_of(Source).to receive(:update_apis)
+    allow(LinkCheckRule).to receive(:create)
```

Comment:
Why are we using these `allow`s like this?

Reply:
Because we want to block the manager making http requests to the worker. There is no way round it.